### PR TITLE
Features/channel configuration

### PR DESCRIPTION
**Pull request by Mikanebu**
* created new temporary google spreadsheet to test
* created new property `gdoc2` in `config.json` 1
* added bot GOOGLE_CLIENT_EMAIL into Google Spreadsheet 
* refactored `main.js` to check for `room` name 
* refactored `bot todos, standups, links` to get appropriate links 
* fixed and wrote unittest
* refactored `message.js` `sendMessage` function to use loop in order to automate

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -2,7 +2,13 @@
     "docs": {
         "gdoc1": {
             "fun": "sendMessage",
-            "dest": "1w4LBF6wbRNVynAk8cQURyz8yZCbTw5hPwcqr87S46hY"
+            "dest": "1w4LBF6wbRNVynAk8cQURyz8yZCbTw5hPwcqr87S46hY",
+            "room": "Datopian"
+        },
+        "gdoc2": {
```
Comment (zelima) [inline config.json]: Is there really good reason naming these `gdoc1` and `gdoc2`. Think would be much useful naming them with reasonable names Eg: `gdocDatopian` and `gdocArtErthTech` etc...
Same with Gist

**Review by zelima — COMMENTED** [review]
Just a minor comment about naming docs.. Let's fix that and think it's ready to go

File: config.json (hunk 1 above)
Comment (Mikanebu) [reply config.json]: ok, will be fixed

Comment (Mikanebu) [reply config.json]: @zelima but in order to distinguish, I added room name. But anyway, I will change it

**Pull request merged**